lib/LedgerSMB/Entity/Note.pm

```diff
@@ -8,7 +8,7 @@ employees, etc.
   @notes = LedgerSMB::Entity::Bank->list($entity_id, [$credit_id]);
   $note->add;
 
-=head1 DESCRIPTION
+=head1 DESCRIPTION 
```

Comment:
It's my preference not to run perltidy on the code base (at least not yet): there will be a break in the 'blame' output, making it harder to trace line  changes back to their original commit.